res/themes/dark/css/_dark.scss

```diff
@@ -42,13 +42,13 @@ $preview-bar-bg-color: $header-panel-bg-color;
 $groupFilterPanel-bg-color: rgba(38, 39, 43, 0.82);
 $inverted-bg-color: $base-color;
 
-$voice-record-stop-border-color: #6F7882; // "Quarterly"
+$voice-record-stop-border-color: #6F7882; // "Quartary"
```

Comment:
Design now agrees we should use the more common spelling "Quaternary". Whatever the word is, I think it would make more sense to create e.g. a `$quaternary-fg-color` and then use that here, instead of just the raw value with a comment.

Reply:
I never expected the rapid iteration to be on this variable of all things...